.github/workflows/release.yaml

```diff
@@ -149,4 +149,33 @@ jobs:
           name: v${{ env.VERSION }}
           draft: true
           prerelease: ${{ startsWith(env.VERSION, '0') }}
-          files: artifacts/*
\ No newline at end of file
+          files: artifacts/*
+
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    needs: [ prepare ]
+    env:
+      VERSION: ${{ needs.prepare.outputs.version }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: https://ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: |
+            ghcr.io/pesde-pkg/registry:latest
```

Comment:
Should only publish to latest if it's not a pre-release.

Reply:
This has been done, not sure if it's the intended way since checking if it starts with a `0` would flag the current version as a pre-release.